include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -61,6 +62,30 @@ class Device {
     Device &operator=(const Device &) = delete;
     Device &operator=(Device &&) = delete;
 
+    /// @brief Check if a certain device extension is available for a specific graphics card.
+    /// @param graphics_card The graphics card.
+    /// @param extension The name of the device extension.s
+    /// @return ``true`` if the requested device extension is available.
+    [[nodiscard]] static bool device_extension_supported(VkPhysicalDevice graphics_card, const std::string &extension);
```

Comment:
Not a huge thing, but I think it would look nicer if the static functions went right under the `public:` specifier and above the constructor.

Reply:
Done.